subt/script/run_kloubak2_jetson.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
is this script only for front jetson? or both? (I am asking about colliding names of script and used configurations)

Reply:
The script is planned for both jetsons. However, i am starting with front one only.